internals/secrethub/inject.go

```diff
@@ -76,15 +78,20 @@ func (cmd *InjectCommand) Run() error {
 		return errio.Error(err)
 	}
 
-	tpl, err := tpl.NewParser("${", "}").Parse(string(raw))
+	varTemplate, err := tpl.NewV1Parser().Parse(string(raw))
```

Comment:
I'm missing something. Didn't we discuss that `v2` would be the default syntax except for `.yml` formatted env files? It seems to me now like `v1` is not only the default but also the only supported one.

Reply:
This PR does not change the inject command. The v2 syntax and variable support still has to be added to the inject command.
As this PR is already pretty big, I propose to add this in a separate PR.

Reply:
This PR also does not add the discussed `--template-version` flag to the run and inject commands.